scripts/browser-pkg

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env node
-var browserify = require('browserify')
-  , path       = require('path')
-  , fs         = require('fs')
+let browserify = require('browserify')
+let path = require('path')
+let = require('fs')
```

Comment:
Should these be `const`?
And isn't `let = require('fs')` a syntax error?

Sorry for the post-merge question, I was interested how you implemented the electron.net support. The lines are still like that in master:

https://github.com/smallwins/slack/blob/82c6430d13d498bbd0cc36c567cf9276b775c812/scripts/browser-pkg#L2-L4

Reply:
Good catch that is an Error!!

WRT to `const`. Nah. If anything they should be `var` in my opinion the new assignment keywords add more confusion than help—as evidenced by the q!

Reply:
imo `const` til you can'st

Reply:
It’s fairly safe now. We had a second order dep breaking builds for older, but still LTS, Node a few months ago. I respect the desire to use new shiny but these features are often breaking, unstable and add little real value. 

There’s something to be said for boring code. It’s usually faster on many facets. Faster to write because you and everyone else knows it. Faster to execute because it’s optimized. Faster to read because the parse step in your brain and the runtime is baked. This faster to debug and maintain. 

Good to question these things really critically and ask what the trade offs really are.